x-pack/test/functional/apps/ml/data_frame_analytics/regression_creation.ts

```diff
@@ -345,6 +372,42 @@ export default function ({ getService }: FtrProviderContext) {
           );
         });
 
+        it('adds discover custom url to the analytics job', async () => {
+          await ml.testExecution.logTestStep('opens edit flyout for discover url');
+          await ml.dataFrameAnalyticsTable.openEditFlyout(testData.jobId);
+          await ml.dataFrameAnalyticsEdit.openEditCustomUrlsForJobTab(testData.jobId);
+
+          await ml.testExecution.logTestStep('add discover custom url for the analytics job');
```

Comment:
Similar to other file, nit suggestion to change to `adds` for consistency with rest of the test.

Reply:
Updated in b8ff7bf3a751c6e7e3c560f4e430c19834df8519